lib/adiwg/mdtranslator/writers/dcat_us/sections/dcat_us_dcat_us.rb

```diff
@@ -80,7 +80,7 @@ def self.build(intObj, responseObj)
                   json.set!('landingPage', landingPage)
                   json.set!('isPartOf', isPartOf)
                   json.set!('systemOfRecords', systemOfRecords)
-                  # json.set!('primaryITInvestmentUII', metadataInfo[:metadataId])
+                  json.set!('primaryITInvestmentUII', metadataInfo[:metadataIdentifier])
```

Comment:
Move the implementation into its own file.

For now, the source path should be metadataInfo.metadataIdentifier.identifier